client/Pages/Home/Home.spec.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+/* eslint-disable react/jsx-key */
+jest.mock('react-i18next');
+import { render } from '@testing-library/react';
+import React from 'react';
+import { Home } from '.';
+import { translate, translateWithFormatting } from '@/utils/test/i18n';
```

Comment:
Could the utils modules become (for example) `test-utils` for these ones and then appropriate names for others? Or does these name get generated by the file structure?

Reply:
at the moment - module mapping is essentially replacing `../../../..` to get to root with `@/` and as such for now the aliases aren't manual but instead based off file path.